package.json

```diff
@@ -12,7 +12,7 @@
     "rotating-file-stream": "^1.3.2",
     "winston": "^3.3.3",
     "winston-daily-rotate-file": "^4.5.5",
```

Comment:
Looks like we need to update this package to the latest too. We're only 1 version behind though: https://github.com/winstonjs/winston-daily-rotate-file/releases